data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,307 @@ mission "Coalition License Patch"
+		conversation
+			`You're immediately hailed by House Mallob when your ship is entering the atmosphere of <origin>. They guide you through the planet, bringing you to what looks like a massive, castle-like structure, which appears to be made out of a similar material to that of Arach ships, sitting at the base of a mountain.`
+			`	You land on a large docking area, where some other ships are also conducting business. Stepping off of your ship you're greeted by what is unmistakingly an outfitter, and as you look around you spot some other large buildings dotting the surrounding area, with members of the Coalition going to and from them. The atmosphere feels like that of a small village turned into an outdoor shopping mall.`
+			`	Shortly after landing you're greeted by an Arach wearing what you judge to be some sort of uniform. He takes a translation device out of his pocket and formally greets you. "Honored to have you as a guest, the great House Mallob is, Captain <first> <last>."`
+			choice
+				`	"Are you the shipwright I'll be taking to Lugglop?"`
+					goto servant
+				`	"What's with this place? Why are there so many buildings all over?"`
+					goto property
+			label servant
+			`	"Mountains, no! But a servant of House Mallob, I am. Mostly as a chauffeur, I work."`
+				goto mistress
+			label property
+			`	"No mere mansion, the private property of House Mallob is," he says as he starts pointing to different areas. "A museum, there we have, detailing the history of Arach ships. To the left, a marketplace is. To the right, House Mallob's private university is, where study the advanced methods of ship making, honor students from all over the Coalition do."`
+			label mistress
+			`	He then asks that you follow him, saying, "Assist Lugglop in his research, my mistress will. Meet her in the mansion, you must." You follow him along the path, and he gives a bit more detail on some of the buildings as you walk near them. What really piques your interest though is that one of the buildings, which he seemed to willingly ignore, had armed guards at the door, with no indication of being members of the Heliarch.`
+			`	Reaching the entrance of the mansion, it isn't as castle-like as the walls, but it all still looks strangely pompous. Up close it almost looks like all the several parts and pods of an Arach Hulk or two were crumpled together into one single building.`
+			`	He guides you through the mansion. You pass by several other servants, notice rooms where he mentions that members of House Mallob are having meetings, and even meet a group of Saryd and Kimek students who came here to study with the shipwrights of House Mallob.`
+			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
+			choice
+				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
+				`	"A Kimek? I thought this was an Arach House."`
```

Comment:
Perhaps have a choice that isn't as joking as these two.

Reply:
added new one